csharp_dotnetcore/samples.ruleset

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'm not a fan of dropping this template ruleset under csharp_core, since we'd need to have another one in csharp_webapi if we'll have webapi samples.

Other options
1) Not have a template and have people copy from other samples
2) Have a template in a folder at the root level where we'll eventually have lint config for python, javascript, etc. Not sure what a good name would be for such a folder. artifacts? build? tools?